core/domain/user_services.py

```diff
@@ -295,6 +296,14 @@ def set_username(user_id, new_username):
     user_settings.username = new_username
     _save_user_settings(user_settings)
 
+def generate_signup_profile_picture(user_id):
```

Comment:
nit, here, above, and below: we keep two blank lines above each top-level function or class definition.